CONTRIBUTING_A_NEW_QUEST.md

```diff
@@ -64,21 +64,21 @@ Use <kbd>Ctrl</kbd>+<kbd>N</kbd> to go to a certain class. <kbd>Ctrl</kbd>+ mous
 Use <kbd>Ctrl</kbd>+<kbd>Q</kbd> (but not on MacOS 😉) after having marked a class, symbol or function to display the documentation for it.
 
 With <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>A</kbd> you can find and explore **any** action you can do in Android Studio (try it by typing for example "go to").
-But "Find usages" (<kbd>alt</kbd>+<kbd>F7</kbd>) is also a very powerful way to find where given function/constant/property/etc is appearing, classified by usages.
+But "Find usages" (<kbd>alt</kbd>+<kbd>F7</kbd>) is also a very powerful way to find where given function/constant/property/etc. appears, classified by usages.
 
 ### Base on existing
 
 Base the new quest on one that exists already.
 
 Find one that has the same type of interface as the one that you are trying to implement.
 
-Are you trying to implement a quest that will have simple yes/no answer? Take [`AddBusStopLit`](app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_lit/AddBusStopLit.kt) quest as a base. Or [`AddTracktype`](app/src/main/java/de/westnordost/streetcomplete/quests/tracktype/AddTracktype.kt) where the mapper will be selecting one of presented images.
+Are you trying to implement a quest that will have a simple yes/no answer? Take the [`AddBusStopLit`](app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_lit/AddBusStopLit.kt) quest as a base. Or [`AddTracktype`](app/src/main/java/de/westnordost/streetcomplete/quests/tracktype/AddTracktype.kt) where the mapper will be selecting from one of some presented images.
 
-Is it going to be asked for POIs and should be disabled by default? [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
+Is it going to be asked for POIs and it should be disabled by default? [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
```

Comment:
```suggestion
Is it going to be asked for POIs and should it be disabled by default? [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
```

Reply:
Actually I don't like the "it" I added now, I think I should revert that line change.

Reply:
> Actually I don't like the "it" I added now, I think I should revert that line change.

Agreed, although as a non-native speaker I always find such multiple-rules (_"is it XXX and should YYYY be ZZZZ?"_)  in question form (instead of if/then form) somewhat hard to parse. 

I.e. something like:
```suggestion
If it is going to be asked for POIs and should be disabled by default, then [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
```

or even better (if the emphasis is supposed to be on disabled-by-default part):

```suggestion
If the quest which is going to be asked for POIs should be disabled by default, then [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
```

would be more readable to me.

Reply:
First one sounded better to me, so I've gone with that